### PR TITLE
Working contact page

### PR DESCRIPTION
**Pull request by vcarl**

### CODE REVIEW

File: src/pages/contact.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ const Index = () => {
           discuss with Reactiflux staff, you can reach out via:
         </p>
         <ul>
+          <li>
+            The public{' '}
+            <a href="https://discord.gg/YKcXEDa">#reactiflux-admin</a>channel
```
Comment (nickserv) [inline src/pages/contact.js]:
```suggestion
            <a href="https://discord.gg/YKcXEDa">#reactiflux-admin</a> channel
```

**Review by nickserv — CHANGES_REQUESTED** [review]
Looks pretty useful overall, just have some formatting/usage suggestions:
- [x] Rename `Message the staff` to `Message the Staff` for consistent casing with other page titles
- [x] We might want to make it more clear that this form isn't for technical help depending on how this gets used
- [x] Code blocks may make links and other formatting harder to read, I think quote blocks (prefixing each line with `> `) would look nicer
- [ ] Perhaps we could rename the bot to something like `Contact Message` (currently `Mod message` which I feel implies that the message is from mods, not for mods)

**Comment by vcarl** [comment]
1) Down to make the change
2) Yeah, should probably make it more prominent that this is for reporting issues
3) Thought about it but the additional value seems low, and it's more complex. it's a textarea, so stripping formatting seems like a good idea, and code block is a pretty easy way to do that
4) I don't have strong feelings

**Comment by nickserv** [comment]
> 3. Thought about it but the additional value [for using code blocks] seems low, and it's more complex. it's a textarea, so stripping formatting seems like a good idea, and code block is a pretty easy way to do that

Is there an advantage to stripping formatting? My understanding is that Discord's sanitization is good enough that it shouldn't be vulnerable to injection attacks. Formatting could be useful such as for clicking links to messages deemed toxic by the user. If you're okay with this I can commit and test it.

```js
const quotedMessage = message.split('\n').map(line => `> ${line}`).join('\n')
```
> 4. I don't have strong feelings [for renaming the bot]

I'd like to try this (and possibly rename the script for consistency), but I don't think I have access to it on Discord's API settings. Please add it to the Reactiflux team if it's not part of an existing bot account.

**Comment by vcarl** [comment]
I'm just thinking that it's more likely to have broken formatting if we don't strip it. If somebody puts in a \`\`\` code block but we're prepending `> ` to all lines, then that's gonna look gnarly.

The message is also only visible to mods, admins, and MVPs, so I don't think there's likely to be confusion about what it's called.

**Comment by nickserv** [comment]
How about posting the message without a code/quote block? The bot name would still make it clear it's from a user, and it makes links clickable without potentially breaking other formatting.

**Pull request merged**